scripts/approve_embargo_terminations.py

```diff
@@ -40,13 +40,13 @@ def main():
     for request in pending_embargo_termination_requests:
         registration = models.Node.find_one(Q('embargo_termination_approval', 'eq', request))
         if not registration.is_embargoed:
-            raise RuntimeError("Registration {0} associated with this embargo termination request ({0}) is not embargoed.".format(
+            logger.warning("Registration {0} associated with this embargo termination request ({0}) is not embargoed.".format(
                 registration._id,
                 request._id
             ))
```

Comment:
You should probably also `continue` here, just to be safe. At the very least, it will prevent incorrect log statements.